core-web/libs/block-editor/src/lib/block-editor.module.ts

```diff
@@ -42,6 +45,10 @@ import { BubbleFormComponent } from './extensions/bubble-form/bubble-form.compon
 import { DotBlockEditorComponent } from './components/dot-block-editor/dot-block-editor.component';
 import { DotEditorCountBarComponent } from './components/dot-editor-count-bar/dot-editor-count-bar.component';
 import { FloatingButtonComponent } from './extensions/floating-button/floating-button.component';
+import { ImageTabviewFormComponent } from './extensions/image-tabview-form/image-tabview-form.component';
+import { DotImageCardListComponent } from './extensions/image-tabview-form/components/dot-image-card-list/dot-image-card-list.component';
+import { DotImageCardComponent } from './extensions/image-tabview-form/components/dot-image-card/dot-image-card.component';
```

Comment:
create a modulo for all these componentes ?

Reply:
Sounds good! I'll do it.